src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -24,25 +27,49 @@ public class DeleteCommand extends Command {
             + "Parameters: -i INDEX (must be a positive integer)\n"
             + "Example: " + SHORT_COMMAND_WORD + " -i 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted: %1$s";
 
-    private final Index targetIndex;
+    private final ArrayList<Index> targetIndexes;
 
-    public DeleteCommand(Index targetIndex) {
-        this.targetIndex = targetIndex;
+    /**
+     * Creates a DeleteCommand to delete the person at the specified {@code Index}.
+     */
+    public DeleteCommand(Index targetindex) {
+        this.targetIndexes = new ArrayList<>();
+        targetIndexes.add(targetindex);
     }
 
+    /**
+     * Creates a DeleteCommand to delete the person at the specified {@code Indexes}.
```

Comment:
Javadocs here could be clearer that you are deleting for multiple people!

Reply:
Updated to improve clarity!